video/out/mac/view.swift

```diff
@@ -35,6 +41,86 @@ class View: NSView, CALayerDelegate {
         wantsBestResolutionOpenGLSurface = true
         wantsExtendedDynamicRangeOpenGLSurface = true
         registerForDraggedTypes([ .fileURL, .URL, .string ])
+        self.autoresizesSubviews = true
+
+        if #available(macOS 13, *) {
+            if ImageAnalyzer.isSupported {
+                analyzer = ImageAnalyzer()
+                analyzerConfig = ImageAnalyzer.Configuration([.text])
+                let overlayView = ImageAnalysisOverlayView()
+                overlayView.preferredInteractionTypes = [.automaticTextOnly]
+                overlayView.autoresizingMask = [.width, .height]
+                overlayView.frame = CGRect(x: 0, y: 0, width: frame.width, height: frame.height)
+                overlayView.isSupplementaryInterfaceHidden = true
+                overlayView.delegate = self
+                analysisOverlayView = overlayView
+                addSubview(overlayView)
```

Comment:
since `ImageAnalysisOverlayView` is added as a subview all the `ImageAnalysisOverlayView` functionality/delegate/as much as possible should be moved into its own view class if possible.